bin/rust/std/lib.rsspec

```diff
@@ -244,58 +243,59 @@ mod sync {
         struct AtomicUsize;
 
         /*@
-        fix value_inner(v: AtomicUsize) -> usize;
-        lem atomic_to_usize_chunk_vs(p: *AtomicUsize);
+        fix AtomicUsize_inner(v: AtomicUsize) -> usize;
+
+        lem AtomicUsize_to_usize(p: *AtomicUsize);
             req *p |-> ?v;
-            ens *(p as *usize) |-> value_inner(v);
+            ens *(p as *usize) |-> AtomicUsize_inner(v) &*& (p as usize) % std::mem::size_of::<usize>() == 0;
 
-        lem usize_to_atomic_chunk_vs(p: *usize);
-            req *p |-> ?v0;
-            ens *(p as *AtomicUsize) |-> ?v &*& value_inner(v) == v0;
+        lem usize_to_AtomicUsize(p: *usize);
```

Comment:
I think this lemma has to be `nonghost_callers_only` to ensure there are no mixed-size races.